README.md

```diff
@@ -105,7 +105,7 @@ Folders corresponding to the scanned subreddit will be created [here](data_colle
 
 Requires a Reddit account with a script application defined. Quick guide available [here](https://towardsdatascience.com/scraping-reddit-data-1c0af3040768)
 
-The credentials for the PRAW connection are fetched from an env file that has to be created and populated manually when running this application for the first time. There is a `.env.template` available [here](.env.template); the three credentials needed for the Reddit application are `CLIENT_ID`, `CLIENT_SECRET`, and `USER_AGENT`.
+The credentials for the PRAW connection are fetched from an env file that has to be created and populated manually when running this application for the first time. There is a `.env.template` available [here](.env.template); the four credentials needed for the Reddit application are `CLIENT_ID`, `CLIENT_SECRET`, `USER_AGENT`, and `REDDIT_RABBITMQ_HOST`.
```

Comment:
Might be worth not putting in environment variables in the `README` and just point them to the `.env.template`, and in the template, comments can be made there. I only say this as environment variables could change, more are added, and it's more overhead to always update the README

Reply:
No